workspaces/quay-actions/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@internal/quay-actions",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
+  },
+  "scripts": {
+    "tsc": "tsc",
+    "tsc:full": "tsc --skipLibCheck true --incremental false",
+    "build:all": "backstage-cli repo build --all",
+    "build:api-reports": "yarn build:api-reports:only",
+    "build:api-reports:only": "backstage-repo-tools api-reports --allow-all-warnings -o ae-wrong-input-file-type --validate-release-tags",
+    "clean": "backstage-cli repo clean",
+    "test": "backstage-cli repo test",
+    "test:all": "backstage-cli repo test --coverage",
+    "fix": "backstage-cli repo fix",
+    "lint": "backstage-cli repo lint --since origin/main",
+    "lint:all": "backstage-cli repo lint",
+    "prettier:check": "prettier --check .",
+    "new": "backstage-cli new --scope @backstage-community",
+    "postinstall": "cd ../../ && yarn install"
+  },
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "plugins/*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/quay-actions"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.28.0",
+    "@backstage/e2e-test-utils": "^0.1.1",
+    "@backstage/repo-tools": "^0.8.0",
```

Comment:
```suggestion
    "@backstage/repo-tools": "^0.10.0",
```

then run `yarn build:api-reports` again

Reply:
I think leaving  `"@backstage/repo-tools": "^0.8.0"`, is fine because otherwise `yarn build:api-reports` outputs braking changes such as the `TypeError: extractorConfig.reportConfigs` is not iterable indicating that the extractorConfig object is not structured as expected, or a property expected to be an array is missing or incorrect.

Reply:
Hmm okay, we'll leave it as is for now then

Reply:
No, this should be updated, please try after following this comment: https://github.com/backstage/community-plugins/pull/1897#discussion_r1834537033